capstone/capapi/documents.py

```diff
@@ -96,11 +110,38 @@ class Meta:
         model = CaseMetadata
         fields = [
             'id',
-            'last_page',
-            'first_page',
             'decision_date',
-            'duplicative',
             'date_added',
         ]
         ignore_signals = True
-        auto_refresh = False
\ No newline at end of file
+        auto_refresh = False
+
+    def full_cite(self):
+        return "%s, %s%s" % (
+            self.name_abbreviation,
+            ", ".join(cite.cite for cite in self.citations),
+            " (%s)" % self.decision_date.year if self.decision_date else ""
+        )
+
+    def get_frontend_url(self, disambiguate=False, include_host=True):
```

Comment:
Does this function ever get called? I don't think it can work. The general context here is that it's hard to generate frontend URLs, because they depend on whether the case shares a citation with any other case. So we (should) only use `CaseMetadata.get_frontend_url()` in situations where we've efficiently figured out whether `disambiguate` should be true, and are using the result to set `CaseMetadata.frontend_url`. Otherwise we don't know if `disambiguate` should be True or False, and we should just use `CaseMetadata.frontend_url`.

Reply:
It works in that it creates the URL, but it was only used in the cite/test_views tests and never got called with the disambiguate flag. I copied it over from CaseMetadata to CaseDocument because it was used in the cite tests, and figured we'd probably start moving some of the functionality over anyway so I'd get the ball rolling.  Easy enough to take out and only use frontend_url!

Reply:
Yeah, I'd say pull it out since it can't really return correct results and I don't want it to be misleading. Could make a simpler test helper if faking something for tests is helpful.

Reply:
Done